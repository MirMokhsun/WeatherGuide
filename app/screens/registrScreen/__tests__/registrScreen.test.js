import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import RegistrScreen from '..';

describe('Registr', () => {
    test('Registr snapshot', () => {
        const renderer = new ShallowRenderer();
        const snap2 = renderer.render(<RegistrScreen />);
        expect(snap2).toMatchSnapshot();
    });
});