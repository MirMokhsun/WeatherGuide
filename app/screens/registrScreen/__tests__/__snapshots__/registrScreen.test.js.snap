// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registr Registr snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#d23844",
    }
  }
>
  <Text>
     registrScreen 
  </Text>
</View>
`;
